_posts/2016-12-19-2016-year-in-review.md

```diff
@@ -0,0 +1,172 @@
+
+```
+open flink-stats/index.html
+```
+2016 is the year that Flink surpassed 1 million lines of code, now clocking in at **1,034,137** lines.
+
+<img src="{{ site.baseurl }}/img/blog/flink-lines-of-code-2016.png" align="center" width="550"
+ alt="Flink Total Lines of Code"/>
+
+Monday remains the day of the week with the most commits over the project's history:
+
+<img src="{{ site.baseurl }}/img/blog/flink-dow-2016.png" align="center" width="550"
+ alt="Flink Commits by Day of Week"/>
+
+And 5pm is still solidly the preferred commit time:
+
+ <img src="{{ site.baseurl }}/img/blog/flink-hod-2016.png" align="center" width="550"
+  alt="Flink Commits by Hour of Day"/>
+
+
+
+
+<br>
+
+### Meetups
+<a href="https://www.meetup.com/topics/apache-flink/" target="_blank">Apache Flink Meetup membership</a> grew by <b>240%</b>
+this year, and at the time of writing, there are 41 meetups comprised of 16,541 members listing Flink as a topic--up from 16 groups with 4,864 members in December 2015.
+The Flink community is proud to be truly global in nature.
+
+<img src="{{ site.baseurl }}/img/blog/flink-meetups-dec2016.png" width="775" alt="Apache Flink Meetup Map"/>
+
+## Flink Forward 2016
+
+The <a href="http://2016.flink-forward.org/" target="_blank">second annual Flink Forward conference </a>took place in
+Berlin on September 12-14, and over 350 members of the Flink community came together for speaker sessions, training,
+and discussion about Flink. <a href="http://2016.flink-forward.org/program/sessions/" target="_blank">Slides and videos</a>
+ from speaker sessions are available online, and we encourage you to take a look if you’re interested in learning more
+ about how Flink is used in production in a wide range of organizations.
+
+Flink Forward will be expanding to <a href="http://sf.flink-forward.org/" target="_blank">San Francisco in April 2017</a>, and the <a href="http://berlin.flink-forward.org/" target="_blank">third-annual Berlin event
+  is scheduled for September 2017.</a>
+
+<img src="{{ site.baseurl }}/img/blog/speaker-logos-ff2016.png" width="775" alt="Flink Forward Speakers"/>
+
+## Features and Ecosystem
+
+### Flink Ecosystem Growth
+
+Flink was added to a selection of distributions during 2016, making it easier
+for an even larger base of users to start working with Flink:
+
+<ul>
+  <li><a href="https://aws.amazon.com/blogs/big-data/use-apache-flink-on-amazon-emr/" target="_blank">
+    Amazon EMR</a></li>
+  <li><a href="https://cloud.google.com/dataproc/docs/release-notes/service#november_29_2016" target= "_blank">
+    Google Cloud Dataproc</a></li>
+  <li><a href="https://www.lightbend.com/blog/introducing-lightbend-fast-data-platform" target="_blank">
+    Lightbend Fast Data Platform</a></li>
+</ul>
+
+In addition, the Flink community built an Apache Beam runner that, according to the Google team, is <a href="https://cloud.google.com/blog/big-data/2016/05/why-apache-beam-a-google-perspective"
```

Comment:
One could also argue that it was the Beam community who built the runner. Maybe rephrase to something like: "The Beam and the Flink communities teamed up to built a Flink runner for Apache Beam that, ..."?